mydumper/mydumper.go

```diff
@@ -263,11 +264,19 @@ func (m *Mydumper) constructArgs() []string {
 	if len(extraArgs) > 0 {
 		ret = append(ret, ParseArgLikeBash(extraArgs)...)
 	}
+	if needToGenerateDoTables(extraArgs) {
+		m.logger.Info("-T/-B/-x index is not given, now we will start to generate table list that mydumper needs to dump through black-white list and route rules from given fromDB")
```

Comment:
1. what is `index` mean in this message?
2. how about remove `and route rules` from the message?

Reply:
What about:
```suggest
m.logger.Info("Tables needed to dump are not given, now we will start to generate table list that mydumper needs to dump through black-white list from given fromDB")
```

Reply:
fine